doc/modules/linear_model.rst

```diff
@@ -24,13 +24,12 @@ To perform classification with generalized linear models, see
 Ordinary Least Squares
 =======================
 
-:class:`LinearRegression` fits a linear model with coefficients
-:math:`w = (w_1, ..., w_p)` to minimize the residual sum
-of squares between the observed targets in the dataset, and the
-targets predicted by the linear approximation. Mathematically it
-solves a problem of the form:
+:class:`LinearRegression` fits a linear model with coefficients :math:`w =
+(w_1, ..., w_p)` to minimize the residual sum of squares between the observed
+targets in the dataset, and the targets predicted by the linear approximation.
+Mathematically it solves a problem of the form:
 
-.. math:: \min_{w} || X w - y||_2^2
+.. math:: \min_{w, w_0} || X w + w_0 - y||_2^2
```

Comment:
We should make it then consistent throughout this document.

Reply:
I agree. Do we introduce the `1_s` vector everywhere for consistency?

Reply:
I would not do it because it is harder to read and most people, I guess, will understand without it.

Reply:
It could be used in a "mathematical details" section.